src/main/java/Backend/Parsers/DateParser.java

```diff
@@ -0,0 +1,89 @@
+package Backend.Parsers;
+
+import java.sql.Time;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class DateParser {
+
+    private LocalDate date;
+    private String dateString;
```

Comment:
Why do you need to store the string format of the date as well? Instead, you can have formatting method (or methods as  required)